metis-authentication/metis-authentication-common/src/main/java/eu/europeana/metis/authentication/user/MetisUser.java

```diff
@@ -71,12 +74,22 @@ public MetisUser() {
    *
    * @param jsonNode the {@link JsonNode} to construct the MetisUser
    * @throws ParseException if the content is unparsable
-   * @throws BadContentException if the content of the JsonNode is unacceptable because of rules that should be followed
+   * @throws BadContentException if the content of the JsonNode is unacceptable because of rules
+   * that should be followed
    */
   public MetisUser(JsonNode jsonNode) throws ParseException, BadContentException {
     parseJsonNodeZohoUserToMetisUser(jsonNode);
   }
 
+  /**
+   * Nullifies any sensitive information of this user.
+   */
+  public void removePrivateDataForPublicUse() {
```

Comment:
I typically don't like if there is a method call required to prepare a database object for being shared. Especially if this is sensitive information: the next person to work on this may forget to add this call.

My solution is usually to make a new object with only the fields you need. But in this case, the MetisUser class already has a JsonIgnore annotation on password... wouldn't that be enough?

Reply:
Removed it for the time being.